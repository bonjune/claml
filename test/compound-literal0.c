struct a {int i;};
b() { &(struct a){0}; }
