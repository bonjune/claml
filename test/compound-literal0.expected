// implicit typedef
// implicit typedef
// implicit typedef
// implicit typedef
// implicit typedef
#line 1 "compound-literal0.c"
struct {
  int i;
} a;
#line 2 "compound-literal0.c"
int b() {
  &(a) {
    { 0, }
  }
}
